wiki/de/de-Installation-for-Linux.md

```diff
@@ -4,22 +4,43 @@ title: "Installation auf Linux"
 lang: "de"
 permalink: "/wiki/Installation-for-Linux"
 ---
+{% include breadcrumb.html root="Jamulus Nutzen" branch1="Erste Schritte" branch1-url="Getting-Started" %}
```

Comment:
Not sure. I think we can change it? Tagging @gilgongo here.

Reply:
Sorry, should have documented the "breadcrumb system" I put in. "root" is a variable, so don't change that or they'll disappear.

Reply:
https://github.com/jamulussoftware/jamuluswebsite/blob/be257dbf0e0770852ff3b49e00aedd52be1b2f40/_includes/breadcrumb.html#L3

shows that it isn't a link but just a title. So changing it shouldn't be dangerous since branch1-url wasn't edited?

Reply:
I'm not sure what you mean by changing, sorry.  If you did:

`{% include breadcrumb.html someThingElse="Jamulus nutzen" branch1="Erste Schritte" branch1-url="Getting-Started" %}`

It would break it by removing the root label. Root is the only thing that's actually compulsory since the H1 of the page is the "leaf". I know there are different conventions for breadcrumbs, but this is a "conservative" approach (and not every page gets a breadcrumb for that reason). 

What you have in the German here is fine (`root="Jamulus nutzen" branch1="Erste Schritte" branch1-url="Getting-Started" `)

Reply:
> It would break it by removing the root label. Root is the only thing that's actually compulsory

Thanks for the answer. This got clear once I looked in the breadcrumb.html file.

> What you have in the German here is fine

Ok. So I'd say, we're ok here.